### ISSUE TITLE
Always use Map for int64 keys

### ISSUE THREAD

**Issue by zhuscat** [stephenh/ts-proto#708]
```
message Foo {
  map<int64, int64> bar = 1;
}
```

is generated to

```ts
export interface Foo {
  bar: { [key: Long]: Long }
}
```

when `--ts_proto_opt=forceLong=long`

This is a type error:

```
An index signature parameter type must be 'string', 'number', 'symbol', or a template literal type.
```

**Comment by stephenh** [comment]
Hey @zhuscat ; thanks for the report. That's a good find.

We already have some code that will use a JS map, which is enabled with:

```
--ts_proto_opt=useMapType=true
```

It would probably be a good idea to _always_ use the map type for longs, even if `useMapType=false`. 

If you'd like to submit a PR for that fix, that'd be great!

Otherwise just using `useMapType=true` should unblock you for now, I think.

**Comment by stephenh** [comment]
This should be fixed by #905 in the next release, thanks @buzap !
